ZipBuilder/Sources/ZipBuilder/LaunchArgs.swift

```diff
@@ -82,6 +86,9 @@ struct LaunchArgs {
     }
   }
 
+  /// The list of architectures to build for.
+  let archs: [String]
```

Comment:
It'd probably be nicer to store this directly as `[Architecture]`, do the unwrapping here (with a failable initializer: `guard let arch = Architecture(rawValue: someString) else { /* fail */ }` then we don't have to force unwrap at the usage point.

Reply:
Done.